cinch/bin/wrappers.py

```diff
@@ -49,10 +49,10 @@ def call_linchpin(work_dir, arg):
     elsewhere
     """
     # cinch will only support a subset of linchpin subcommands
-    supported_cmds = ['rise']
+    supported_cmds = ['rise', 'drop']
     if arg not in supported_cmds:
-        raise Exception('linchpin command \'' + arg + '\' not '
-                        'supported by cinch')
+        raise Exception('linchpin command "%s" not '
+                        'supported by cinch' % arg)
```

Comment:
This syntax is deprecated heading towards Python 3. "some string{0}".format(arg) is preferred.